db/db_impl.h

```diff
@@ -803,6 +803,25 @@ class DBImpl : public DB {
   // Additonal options for compaction and flush
   EnvOptions env_options_for_compaction_;
 
+  std::unique_ptr<ColumnFamilyMemTablesImpl> column_family_memtables_;
+
+  FlushScheduler flush_scheduler_;
```

Comment:
It seems that `flush_scheduler_` does not need to be protected, at least for now it can remain private.